### PR TITLE
Update toolbar layout

### PR DESCRIPTION
**Pull request by zBritva**

### CODE REVIEW

**Comment by zBritva** [comment]
@danmarshall Could you review the PR? Thanks!

**Comment by danmarshall** [comment]
Can you wait for #379 , there will probably be a merge conflict.

File: src/app/main_view.tsx [hunk 1]
```diff
@@ -83,10 +87,14 @@ export class MainView extends React.Component<MainViewProps, MainViewState> {
   }
 
   public render() {
-    const toolBarCreator = (layout: "vertical" | "horizontal") => {
+    const toolBarCreator = (config: {
+      undoRedoLocation: "menubar" | "toolbar"
```
Comment (danmarshall) [inline src/app/main_view.tsx:91]: Can we use enums here? I'd like to see much less strings in the codebase

File: src/app/views/menubar.tsx [hunk 2]
```diff
@@ -553,7 +542,7 @@ export class MenuBar extends ContextedComponent<
             ) : null}
           </div>
           <div className="charticulator__menu-bar-center el-text">
-            <p>{this.context.store.chart?.properties.name} - Charticualtor</p>
+            <p>{this.context.store.chart?.properties.name} - Microsoft Charticualtor</p>
```
Comment (danmarshall) [inline src/app/views/menubar.tsx:545]: This will be in the new lang strings

**Review by danmarshall — CHANGES_REQUESTED** [review]
Sorry about the merge conflicts, I was trying to avoid them!

**Comment by zBritva** [comment]
@danmarshall is it ok now?

File: src/app/views/tool_bar.tsx [hunk 3]
```diff
@@ -38,11 +38,276 @@ export class Toolbar extends ContextedComponent<
...
+        <span className={"chartaccent__toolbar-horizontal-separator"}/>
+          {labels && (
+            <span
+              className={
+                this.props.layout === "vertical"
+                  ? "chartaccent__toolbar-vertical-label"
+                  : "chartaccent__toolbar-label"
+              }
+            >
+              Marks
```
Comment (danmarshall) [inline src/app/views/tool_bar.tsx:54]: strings.toolbar.marks

File: src/app/views/tool_bar.tsx [hunk 4]
```diff
@@ -38,11 +38,276 @@ export class Toolbar extends ContextedComponent<
...
+            </span>
+          )}
+          <MultiObjectButton
+            compact={this.props.layout === "vertical"}
+            tools={[
+              {
+                classID: "mark.rect",
+                title: "Rectangle",
```
Comment (danmarshall) [inline src/app/views/tool_bar.tsx:62]: these titles should come from strings

File: src/app/views/tool_bar.tsx [hunk 5]
```diff
@@ -38,11 +38,276 @@ export class Toolbar extends ContextedComponent<
...
+            classID="mark.nested-chart"
+            title="Nested Chart"
+            icon="mark/nested-chart"
+          /> */}
+          {this.props.undoRedoLocation === UndoRedoLocation.ToolBar ? 
+          (
+            <>
+              <span className={"chartaccent__toolbar-horizontal-separator"}/>
+              <ToolButton
+                title="Undo (Ctrl-Z)"
```
Comment (danmarshall) [inline src/app/views/tool_bar.tsx:139]: title from strings

File: src/app/views/tool_bar.tsx [hunk 6]
```diff
@@ -575,14 +788,17 @@ export class ScaffoldButton extends ContextedComponent<
...
+  label: boolean
+}, {}> {
   public container: HTMLSpanElement;
 
   public render() {
     return (
       <span ref={(e) => (this.container = e)}>
         <ToolButton
-          title={strings.toolbar.link}
+          title="Link"
```
Comment (danmarshall) [inline src/app/views/tool_bar.tsx:800]: strings

**Review by danmarshall — CHANGES_REQUESTED** [review]

**Review by danmarshall — APPROVED** [review]

**Pull request merged**